test.bash

```diff
@@ -4,7 +4,7 @@
 trap 'kill $(jobs -pr)' SIGINT SIGTERM EXIT
 
 echo "Doing an early build"
-cargo build --workspace --tests 
+cargo build --bin lite-rpc --tests
```

Comment:
build workspace. why just lite-rpc ?

Reply:
I wanted test.bash to. do the integration test with running solana validator - the cargo build+test is done by the other workflow. maybe I should rename _test.bash_ to sth like _test_with_validator.bash_ or so.